core/templates/dev/head/pages/exploration_editor/exploration_metadata_form_directive.html

```diff
@@ -0,0 +1,70 @@
+<p>
+  Add a title: what should your exploration be called?
+  <input id="explorationTitle" type="text" class="form-control protractor-test-exploration-title-input" ng-model="explorationTitleService.displayed" ng-blur="saveExplorationTitle()" placeholder="Choose a title for your exploration." focus-on="<[::EXPLORATION_TITLE_INPUT_FOCUS_LABEL]>" maxlength="40" ng-trim="false">
+  <span class="help-block" style="font-size: smaller">
+    <em>Please use at most 40 characters, so that this fits in the summary card.</em>
+    <span ng-if="explorationTitleService.displayed.length > 0">
+      <em>Characters used: <[(explorationTitleService.displayed.length)]>/40</em>
+    </span>
+  </span>
+</p>
+<form name="objectiveForm">
+  <p ng-class="{'has-error': (objectiveForm.objectiveInput.$touched || objectiveHasBeenPreviouslyEdited) && (explorationObjectiveService.displayed.length < 15) || (explorationObjectiveService.displayed.length > 100)}">
+    Add a goal: what does the exploration help people to do?
+    <input id="explorationObjective" type="text" class="form-control protractor-test-exploration-objective-input"
+    ng-model="explorationObjectiveService.displayed" ng-blur="saveExplorationObjective()" placeholder="Learn how to..." ng-trim="false">
+    <span class="help-block" style="font-size: smaller"> 
+      <em>In a complete sentence, tell people what they'll learn from this exploration.</em>
+      <span ng-if="explorationObjectiveService.displayed.length > 0">
+        <em>Characters used: <[(explorationObjectiveService.displayed.length)]>/100</em>
+      </span>
+    </span>
+  </p>
+</form>
+<p ng-if="explorationInitStateName">
```

Comment:
I guess something about this ng-if isn't working because it's not displaying this field.